conf.py

```diff
@@ -108,7 +108,7 @@
     'kotlin-sdk': ('https://docs.mongodb.com/realm-sdks/kotlin/10.0.0-BETA.6/%s', ''),
     'swift-sdk': ('https://docs.mongodb.com/realm-sdks/swift/10.0.0-beta.5/%s', ''),
     'objc-sdk': ('https://docs.mongodb.com/realm-sdks/objc/10.0.0-beta.5/%s', ''),
-    'js-sdk': ('https://docs.mongodb.com/realm-sdks/js/10.0.0-beta.9/%s', ''),
+    'js-sdk': ('https://docs.mongodb.com/realm-sdks/js/10.0.0-beta.13/%s', ''),
```

Comment:
```suggestion
    'js-sdk': ('https://docs.mongodb.com/realm-sdks/js/%s', ''),
```

will default to the latest version